catalog/dags/data_refresh/data_refresh_task_factory.py

```diff
@@ -101,14 +101,20 @@ def create_data_refresh_task_group(
 
     target_alias = data_refresh.media_type  # TODO: Change when using versioned aliases
 
+    # Retrieve the correct poke amount for media_type
+    poke_interval = REFRESH_POKE_INTERVAL
```

Comment:
We receive the `DataRefresh` config object in this factory method as `data_refresh`, so we can actually just access the poke interval directly as `data_refresh.poke_interval`, instead of iterating through the configs.

Reply:
@stacimc The explanation is incredible. I have added the `data_refresh_poke_interval` and `filtered_index_poke_interval` options to the `DataRefresh` type.

Reply:
Great :) I got the timings of the operations we're waiting on by looking at production runs in Airflow, which does require prod access.